src/test/scala/nl.knaw.dans.easy.sword2/TestSupportFixture.scala

```diff
@@ -29,4 +29,22 @@ trait TestSupportFixture extends FlatSpec with Matchers with OptionValues {
     path.createDirectories()
     path
   }
+
+  def createMinimalSettings(bag: File): SwordConfig = {
+    new SwordConfig {
+      settings = Settings(depositRootDir = bag.toJava,
+        "rwxrwxrwx",
+        new java.io.File("dummy"),
+        "",
+        "",
+        null,
+        null,
+        None,
+        "support@dans.knaw.nl",
```

Comment:
use something like `"does.not.exists@dans.knaw.nl"`, as @jo-pol introduced recently...